src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -155,6 +168,30 @@ public String apply(final String str) {
                     )
                 )
             );
+        EndsDaemon.deprecate(directives, deprecate);
+        return directives;
+    }
+
+    /**
+     * Adds the deprecated message to the directive.
+     * @param directives the directives which being deprecated
+     * @param deprecate true if it is deprecated.
+     */
+    private static void deprecate(final Directives directives,
+        final boolean deprecate) {
+        if (deprecate) {
+            directives.add("deprecate").set(
+                "#### Deprecation Notice ####\n" +
```

Comment:
@mkordas I don't think, it is duplicated code. The message in the start and end are not same.